### PR TITLE
doc: added doc.txt to serialization test

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description

the doc.txt was added to the serialization test module where it is explained  how to execute it

### CODE REVIEW

**Review by eduazocar — COMMENTED** [review]
Fix only the request changes of @luisan00

File: tests/serialization/doc.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Here the unit tests are carried out on the serialization module.
+
+To test the code in automatic mode:
+
+```c
+make flash test
+
+```
+
+To test the code in interactive mode:"
```
Comment (eduazocar) [inline tests/serialization/doc.txt:18]: remove this "

Comment (eduazocar) [inline tests/serialization/doc.txt:18]: remove this "

**Review by eduazocar — APPROVED** [review]
That's alteady :+1:

**Review by luisan00 — APPROVED** [review]

**Pull request merged**